### DISCUSSION TITLE
client-java-core & client-java-jersey java compatibility

### DISCUSSION THREAD

**Discussion by pontamilselvan** [featurehub-io/featurehub#541] [Q&A] [answered]
are client-java-core & client-java-jersey java compatible with java 1.8?

**Comment by rvowles** [comment]
yes - they are both built and released with java 8

**Comment by rvowles** [comment, marked as answer]
the version I am referring to is java-client-jersey 2.2 - 1.2 was only released for Java 11.

**Reply by pontamilselvan** [reply]
sorry i didnt mention the versions earlier...

i was asking abt client-java-core 2.3 & client-java-jersey 2.2 ? (these two comaptible with jdk 1.8?)

**Reply by rvowles** [reply]
2.3 and 2.2 are JDK 8 compatible. Core needs for be for Android support :-)
